gnome/gnome-vfsmm/Portfile

```diff
@@ -1,29 +1,38 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem       1.0
+PortSystem      1.0
```

Comment:
This commit contains both functional changes and whitespace changes to the remainder of the file; this is difficult to review or analyze. Please move substantial whitespace-only changes into a separate commit.

Reply:
I made a similar comment above.

Reply:
Ok, will do

Reply:
@pmetzger @ryandesign Separated for each port.